<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSaveRenameSettings</class>
 <widget class="QDialog" name="DialogSaveRenameSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save current rename setting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Settings list</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="SavedSettingsListWidget" name="listWidget"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>6</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>New setting name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit">
     <property name="text">
      <string>New setting</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>6</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOK">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SavedSettingsListWidget</class>
   <extends>QListWidget</extends>
   <header>widgets/SavedSettingsListWidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>listWidget</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>buttonOK</tabstop>
  <tabstop>buttonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOK</sender>
   <signal>clicked()</signal>
   <receiver>DialogSaveRenameSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>DialogSaveRenameSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DialogSaveRenameSettings</receiver>
   <slot>onLineEditTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>34</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>itemActivated(QListWidgetItem*)</signal>
   <receiver>buttonOK</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>itemPressed(QListWidgetItem*)</signal>
   <receiver>DialogSaveRenameSettings</receiver>
   <slot>onListWidgetItemPressed(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>buttonOK</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>DialogSaveRenameSettings</receiver>
   <slot>onListWidgetCurrentTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onLineEditTextChanged(QString)</slot>
  <slot>onListWidgetItemPressed(QListWidgetItem*)</slot>
  <slot>onListWidgetCurrentTextChanged(QString)</slot>
 </slots>
</ui>
